### DISCUSSION TITLE
Matter 1.1 specification and Matter 1.1 SDK released, brings enhancements for developers and devices

### DISCUSSION THREAD

**Discussion by Hedda** [matter-js/matter.js#153] [General]
FYI, Matter 1.1 specification and Matter 1.1 SDK was released yesterday and is said to bring "**_enhancements for developers and devices_**":

https://csa-iot.org/newsroom/matter-1-1-release-enhancements-for-developers-and-devices/

Matter 1.1 specification is available for download by request:

https://csa-iot.org/developer-resource/specifications-download-request/

Matter 1.1 SDK is available via GitHub (connectedhomeip repository under the project-chip organization):

https://github.com/project-chip/connectedhomeip/tree/v1.1.0.1

Highlights:

* _Enhanced support for battery-operated devices._
  *  _Matter 1.1 enhances support for a category of devices that apply to many smart home products — Intermittently Connected Devices (ICDs). Sometimes called “sleepy devices,” these are typically battery-powered devices like contact, motion, and temperature sensors as well as door locks and switches that need to conserve power for optimal operation and lifespan. The additional support reduces the likelihood that a device will be reported as offline when users or platforms interact with it. These improvements mean developers will find it easier to optimize their products and create better user experiences._ 

* _One of the benefits and differentiators of Matter is its publicly available specification and open-source SDK that allows all developers to work from a common specification and code base. Of course, it’s important that those tools are easy to work with, so we’ve taken feedback and learning from members and early developers, and made clarifications and clean-up to the specification. This will make it easier for developers to dive in, understand, and use the specification. It’s also easier for developers to contribute to the Matter specification, with better guidance for contributing support for new device types._

**Comment by Hedda** [comment]
Matter 1.1 does not support any new device types which is both surprising and a little sad as such was expected (with Matter 1.0 supporting so few device types when compared to the latest Zigbee Cluster Library). Matter 1.1 does at least contain many bug fixes and foundational improvements under the hood so hopefully also upstream Matter 1.x is becoming more stable.

**Comment by Apollon77** [comment]
In fact we support Matter 1.1 fully right now
